test/parallel/test-child-process-spawnsync-shell.js

```diff
@@ -52,16 +53,18 @@ assert.strictEqual(env.stdout.toString().trim(), 'buzz');
     const shellFlags = platform === 'win32' ? ['/d', '/s', '/c'] : ['-c'];
```

Comment:
I don't think it really matters much since we're already explicitly passing `'win32'` to this function below here.